src/test/java/uk/gov/hmcts/reform/professionalapi/controller/internal/ProfessionalUserInternalControllerTest.java

```diff
@@ -83,14 +83,67 @@ public void testFindUsersByOrganisation() {
         verify(professionalUserServiceMock, times(1)).findProfessionalUsersByOrganisation(organisation, "true", true, "");
     }
 
+
+    @Test
+    public void testFindUsersByOrganisationwithoutRoles() {
+        final HttpStatus expectedHttpStatus = HttpStatus.OK;
+        ProfessionalUser professionalUser = new ProfessionalUser("fName", "lastName", "emailAddress", organisation);
+
+        List<SuperUser> users = new ArrayList<>();
+        users.add(professionalUser.toSuperUser());
+        organisation.setUsers(users);
+        organisation.setStatus(OrganisationStatus.ACTIVE);
+
+        when(organisationServiceMock.getOrganisationByOrgIdentifier(organisation.getOrganisationIdentifier())).thenReturn(organisation);
+        when(professionalUserServiceMock.findProfessionalUserProfileByEmailAddress("emailAddress")).thenReturn(professionalUser);
+        when(professionalUserServiceMock.findProfessionalUsersByOrganisation(any(Organisation.class), any(String.class), any(Boolean.class), any(String.class))).thenReturn(responseEntityMock);
+        when(responseEntityMock.getStatusCode()).thenReturn(HttpStatus.OK);
+
+        doNothing().when(organisationIdentifierValidatorMock).validate(any(Organisation.class), any(OrganisationStatus.class), any(String.class));
+        doNothing().when(organisationCreationRequestValidatorMock).validateOrganisationIdentifier(any(String.class));
+
+        ResponseEntity<?> actualRolesFalse = professionalUserInternalController.findUsersByOrganisation(organisation.getOrganisationIdentifier(), "true", true, null, null);
+        assertThat(actualRolesFalse).isNotNull();
+        assertThat(actualRolesFalse.getStatusCode().value()).isEqualTo(expectedHttpStatus.value());
+
+        verify(organisationServiceMock, times(1)).getOrganisationByOrgIdentifier(organisation.getOrganisationIdentifier());
+        verify(professionalUserServiceMock, times(1)).findProfessionalUsersByOrganisation(organisation, "true", true, "");
+    }
+
+    @Test
+    public void testFindUsersByOrganisationwithoutRolesDefaultNull() {
+        final HttpStatus expectedHttpStatus = HttpStatus.OK;
+        ProfessionalUser professionalUser = new ProfessionalUser("fName", "lastName", "emailAddress", organisation);
+
+        List<SuperUser> users = new ArrayList<>();
+        users.add(professionalUser.toSuperUser());
+        organisation.setUsers(users);
+        organisation.setStatus(OrganisationStatus.ACTIVE);
+
+        when(organisationServiceMock.getOrganisationByOrgIdentifier(organisation.getOrganisationIdentifier())).thenReturn(organisation);
+        when(professionalUserServiceMock.findProfessionalUserProfileByEmailAddress("emailAddress")).thenReturn(professionalUser);
+        when(professionalUserServiceMock.findProfessionalUsersByOrganisation(any(Organisation.class), any(String.class), any(Boolean.class), any(String.class))).thenReturn(responseEntityMock);
+        when(responseEntityMock.getStatusCode()).thenReturn(HttpStatus.OK);
```

Comment:
Please add a verify for this condition

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/612/commits/acbe4881fa2e5d46be921bc81e0a93e8f8bf9f4f